resources/fixtures/config/barcode.yml

```diff
@@ -1,3 +1,9 @@
 file-type: 'png'
 barcode-type: 'code39'
+
+# determines the class that will be used to automatically assign barcodes to products, will typically be the same as
+# the barcode-type
+barcode-generator: 'code39'
```

Comment:
Makes sense, could it have a default value? So in the case where you just want to generate them as ean-13 with default MS values you only have to change one thing?